packages/transaction_signing_gateway/lib/model/network_info.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:equatable/equatable.dart';
+
+class NetworkInfo extends Equatable {
```

Comment:
we can remove that after suggested changes are in place

Reply:
✅